package-lock.json

```diff
@@ -9707,7 +9718,9 @@
         "get-port": "^7.1.0"
       },
       "devDependencies": {
-        "@netlify/types": "2.0.2"
+        "@netlify/types": "2.0.2",
+        "tsup": "^8.0.0",
+        "vitest": "^3.0.0"
```

Comment:
These packages added here were already used, but they were unlisted in the respective package.json files